webapp/logging.json

```diff
@@ -20,6 +26,9 @@
     "loggers": {
         "gunicorn": {
             "level": "DEBUG"
+        },
+        "sqlalchemy.engine": {
+            "level": "INFO"
```

Comment:
This part says that we log everything in `SQLAlchemy.engine.level = Info`, correct?

Reply:
Pretty much. It says that the logger named `"sqlalchemy.engine"` (i.e. the one you get, when you call `logging.getLogger("sqlalchemy.engine")`) should log all messages at level `INFO` and above.